config.go

```diff
@@ -51,6 +51,9 @@ type config struct {
 
 	// DisableGraph disables collection of graph metrics
 	DisableGraph bool `long:"disablegraph" description:"Do not collect graph metrics"`
+
+	// DisableHtlc disables collection of HTLCs metrics
```

Comment:
nit: Bad example on the comment above, but we prefer full sentences for comments, including punctuation.

Reply:
Noted! 497ed32

Reply:
Sorry, I mostly meant the missing full stop :sweat_smile: Feel free to correct that for the existing comment above as well.
And can you please squash the fixup commit with the one that added the line? And usually we prefix the commits with the package that is changed (though not super strictly enforced in this repo): https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md